jujugui/static/gui/src/app/components/header-help/header-help.js

```diff
@@ -58,9 +58,10 @@ class HeaderHelp extends React.Component {
   _generateIssuesLink() {
     let label = 'File Issue';
     let link = 'https://github.com/juju/juju-gui/issues';
-    if (this.props.gisf && this.props.user) {
+    if (this.props.user) {
       label = 'Get Support';
```

Comment:
I'm not sure that this label makes sense in the gijoe case. Maybe "Documentation" is better in that case?